experiments/modern/src/runtime/PopupMenu.js

```diff
@@ -0,0 +1,22 @@
+const GuiObject = require("./GuiObject");
+
+class PopupMenu extends GuiObject {
+  /**
+   * getClassName()
+   *
+   * Returns the class name for the object.
+   * @ret The class name.
+   */
+  static getClassName() {
+    return "PopupMenu";
+  }
+  addCommand(txt, id, checked, disabled) {
+  }
```

Comment:
We should create a utility called something like `unimplemented` which takes a string and just wraps `console.warn`.

That will make it easy to see all these places where we're stubbing things out. We'll be able to grep for them, and also see them at runtime. If they get too noisy we can easily turn them off by default and require a flag to enable them or something.

Reply:
👍  Yes, definitely need to keep track of these